v63004/u_inref/gtm8202.csh

```diff
@@ -0,0 +1,131 @@
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~1,2,3,4,5,6 "
+echo "#  expected to return sequences 2,3,4,5"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=~1,2,3,4,5,6 "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~1,2,~3"
+echo "#  expected to return sequences 2"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=~1,2,~3 "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~(~1,2,~3)"
+echo "#  expected to return sequences 1,3"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno="~(~1,2,~3)" "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=1,~2,3"
+echo "#  expected to return sequences 1,3"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=1,~2,3 "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  multi region extract -SEQNO=~(1,~2,3)"
+echo "#  expected to return sequences 2"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno="~(1,~2,3)" "*" >>& db_log.txt
+echo "#  search extract file for set variables"
+if (  -f Reg.mjf ) then
+	$grep "=" Reg.mjf | awk -F '\\' '{ print $6 " " $11 }'
+	rm Reg.mjf
+else
+	echo "Reg.mjf not created (not supposed to happen here)"
+endif
+
+echo "#  single region extract -SEQNO=1,2,3,4,5,6,7,8 TP transaction "
+echo "#  expecting a.lost to be created"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno=1,2,3,4,5,6,7,8 "a.mjl" >>& db_log.txt;
+if (  -f "a.lost" ) then
+	echo "a.lost file created (expected)"
+else
+	echo "No a.lost file created (NOT expected)"
+endif
+
+
+echo "#  shutdown replication and turn it back on for AREG"
+$MUPIP replic -source -shutdown -timeout=0 >>& passive_stop.out
+$gtm_tst/com/dbcheck.csh >> dbcreate_log.txt
+$MUPIP set -repli=on  -reg "AREG" >>& db_log.txt
+$MUPIP set -repli=off  -reg "BREG" >>& db_log.txt
+$MUPIP set -repli=off  -reg "DEFAULT" >>& db_log.txt
+
+echo "#  multi region extract -SEQNO=1,2,3,4 with BREG replication turned off"
+echo "#  expecting JNLEXTRCTSEQNO error"
+$MUPIP journal -forward -extract=./Reg.mjf -seqno="1,2,3,4" "*" >>& db_log.txt
```

Comment:
The test is expecting a JNLEXTRCTSEQNO error. But the presence of that error needs to be tested here.

Reply:
I am fine with all changes. The only one change missing is the test of unreplicated regions with -SEQNO on a single region using database transaction number.